### PR TITLE
add styles for header

### PR DESCRIPTION
**Pull request by KonstantinLigus**
с почином))

### CODE REVIEW

File: src/types/ILink.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { ReactNode } from 'react'
```
Comment (45s) [inline src/types/ILink.ts:1]: please, move it closer to component `ui/link/types.ts`

File: src/ui/Header/template.tsx [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+        </Link>
+        <Link to="/prices" className={styles.link}>
+          Prices
+        </Link>
+        <Link to="/history" className={styles.link}>
+          History
+        </Link>
+      </nav>
+    </header>
+  )
```
Comment (45s) [inline src/ui/Header/template.tsx:23]: it's not critical, only suggestion:

```
const links: [string, string][] = [
  ['/', 'Dashboard'],
  ['/prices', 'Prices'],
  ['/history', 'History'],
  ]
  
  

function Header () {

return (<header>
  {links.map(([path, label]) => <Link> 
      key={path}
      to={path}
      className={styles.link}
      >
       {label}
       
<Link>  
      

</header>)

}
  
```

**Review by 45s — APPROVED** [review]
nice work!, thanks!

File: src/ui/Header/template.tsx (hunk 2 above)
Comment (KonstantinLigus) [reply src/ui/Header/template.tsx:23]: Maybe, is it better move `const links` to another file?

File: src/types/ILink.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+import { NavLinkProps } from 'react-router-dom'
+
+export type ILink = {
```
[the remark this answers is not in the document]
Comment (KonstantinLigus) [reply src/types/ILink.ts]: Done

File: src/ui/Header/template.tsx (hunk 2 above)
Comment (KonstantinLigus) [reply src/ui/Header/template.tsx:21]: Done

Comment (45s) [reply src/ui/Header/template.tsx:21]: yep, you are right

File: src/types/ILink.ts (hunk 1 above)
Comment (KonstantinLigus) [reply src/types/ILink.ts]: Done

**Pull request merged**